.github/workflows/stylish-haskell.yml

```
@@ -38,86 +38,18 @@ jobs:
-        cabal build all --dry-run
-
-    # For users who fork cardano-node and want to define a writable cache, then can set up their own
-    # S3 bucket then define in their forked repository settings the following secrets:
-    #
-    #   AWS_ACCESS_KEY_ID
-    #   AWS_SECRET_ACCESS_KEY
-    #   BINARY_CACHE_URI
-    #   BINARY_CACHE_REGION
-    - name: Cabal cache over S3
-      uses: action-works/cabal-cache-s3@v1
-      env:
-        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      with:
-        region: ${{ secrets.BINARY_CACHE_REGION }}
-        dist-dir: stylish-haskell-${{ env.STYLISH_HASKELL_VERSION }}/dist-newstyle
-        store-path: ${{ steps.setup-haskell.outputs.cabal-store }}
-        threads: 16
-        archive-uri: ${{ secrets.BINARY_CACHE_URI }}/${{ env.CABAL_CACHE_VERSION }}/${{ runner.os }}
-        skip: "${{ secrets.BINARY_CACHE_URI == '' }}"
-
-    # It's important to ensure that people who fork this repository can not only successfully build in
-    # CI by default, but also have meaning cabal store caching.
-    #
-    # Because syncing with S3 requires credentials, we cannot rely on S3 for this. For this reason a
-    # https fallback is used. The https server mirrors the content of the S3 bucket. The https cabal
-    # store archive is read-only for security reasons.
-    #
-    # Users who fork this repository who want to have a writable cabal store archive are encouraged
-    # to set up their own S3 bucket.
-    - name: Cabal cache over HTTPS
-      uses: action-works/cabal-cache-s3@v1
-      with:
-        dist-dir: stylish-haskell-${{ env.STYLISH_HASKELL_VERSION }}/dist-newstyle
-        store-path: ${{ steps.setup-haskell.outputs.cabal-store }}
-        threads: 16
-        archive-uri: https://iohk.cache.haskellworks.io/${{ env.CABAL_CACHE_VERSION }}/${{ runner.os }}
-        skip: "${{ secrets.BINARY_CACHE_URI != '' }}"
-        enable-save: false
-
-    - name: Build stylish-haskell
-      run: |
-        cd "stylish-haskell-$STYLISH_HASKELL_VERSION"
-        cabal build all
-
-    - name: Install stylish-haskell
       run: |
-        cd "stylish-haskell-$STYLISH_HASKELL_VERSION"
-        cabal install exe:stylish-haskell
+        version="${{ env.STYLISH_HASKELL_VERSION }}"
 
-    - name: Remove stylish-haskell build directory
-      run: rm -rf "stylish-haskell-$STYLISH_HASKELL_VERSION"
+        curl -sL \
+          "https://github.com/haskell/stylish-haskell/releases/download/v$version/stylish-haskell-v$version-linux-x86_64.tar.gz" \
+          > /tmp/stylish-haskell.tar.gz
 
-    - name: stylish-haskell installation directory
-      run: find . -name stylish-haskell
+        tar -C "/tmp" -zxvf /tmp/stylish-haskell.tar.gz
 
-    - name: Add installation directory to path
-      run: echo "PATH=$HOME/.cabal/bin:$PATH" >> $GITHUB_ENV
+        mv "/tmp/stylish-haskell-v$version-linux-x86_64/stylish-haskell" /usr/local/bin
```

Is this needed? Just set `PATH=/tmp/stylish-haskell-v$version-linux-x86_64:$PATH` instead?
```suggestion
        echo "PATH=/tmp/stylish-haskell-v$version-linux-x86_64:$PATH" >> $GITHUB_ENV
```